config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -44,22 +44,6 @@ presubmits:
       testgrid-dashboards: presubmits-test-infra
       testgrid-tab-name: gubernator
 
-  - name: pull-test-infra-yamllint
```

Comment:
The reason this was an extra job is that this needs `yamllint` in the image that runs the job. Does the image that runs `verify/all.sh` have `yamllint` installed?

Reply:
verify/all.sh shells out to verify/yamllint.sh, which does a docker run, so basically this uses DinD